cli/internal/cmd/upgradeapply.go

```diff
@@ -369,10 +370,7 @@ func (u *upgradeApplyCmd) confirmIfUpgradeAttestConfigHasDiff(cmd *cobra.Command
 			return errors.New("aborting upgrade since attestation config is different")
 		}
 	}
-	// TODO(elchead): move this outside this function to remove the side effect.
-	if err := u.kubeUpgrader.BackupConfigMap(cmd.Context(), constants.JoinConfigMap); err != nil {
-		return fmt.Errorf("backing up join-config: %w", err)
-	}
+
```

Comment:
Is this done somewhere else now?

Reply:
Yes, see: https://github.com/edgelesssys/constellation/blob/14b470192fc09f1f101f8c1fac79b7b92a7ae2ba/cli/internal/kubecmd/kubecmd.go#L186-L216